doc/classes/Animation.xml

```diff
@@ -271,12 +279,60 @@
 				Returns the index of the specified track. If the track is not found, return -1.
 			</description>
 		</method>
+		<method name="get_marker_at_time" qualifiers="const">
+			<return type="StringName" />
+			<param index="0" name="time" type="float" />
+			<description>
+				Returns the name of the marker located at the given time.
+			</description>
+		</method>
+		<method name="get_marker_color" qualifiers="const">
+			<return type="Color" />
+			<param index="0" name="name" type="StringName" />
+			<description>
+				Returns the given marker's color.
+			</description>
+		</method>
+		<method name="get_marker_names" qualifiers="const">
+			<return type="PackedStringArray" />
+			<description>
+				Returns every marker in the Animation, sorted ascending by time.
```

Comment:
```suggestion
				Returns every marker in this Animation, sorted ascending by time.
```
Might be good to do in general later but "the Animation" to me implies that an animation is provided, not that it means this animation